test/beast/core/file_test.hpp

```diff
@@ -35,16 +36,38 @@ test_file()
 
     namespace fs = boost::filesystem;
 
+    static constexpr
+#ifdef _WIN32
+    boost::winapi::WCHAR_ unicode_suffix[] = { 0xd83e, 0xdd84, 0x0000 }; // UTF-16-LE unicorn
+#else
+    char                  unicode_suffix[] = { 0xf0, 0x9f, 0xa6, 0x84, 0x00 }; // UTF-8 unicorn
```

Comment:
```suggestion
    const char* unicode_suffix = "\xf0\x9f\xa6\x84"; // UTF-8 unicorn
```

Reply:
This would also solve the narrowing error in clang.
This PR is now superceded by PR1793, in which I fixed the error by using char literals.
Not as elegant as this solution, but I wanted to make the smallest change possible.